examples/examples_nodejs_test.go

```diff
@@ -301,17 +301,18 @@ func TestAccEcr(t *testing.T) {
 }
 
 func TestAccDeleteBeforeCreate(t *testing.T) {
+	basePath := filepath.Join(getCwd(t), "delete_before_create", "mount_target")
```

Comment:
Why is this needed? In go relative paths seem to resolve relative to the test. Is there something that's now requiring getCwd prefix? Can we roll back that something?

Reply:
Ah I see = never mind. Previously "step2" was interpreted as ./step2 but that was missing. After the change it's interpreted as "$PWD/delete_before_create/mount_target/step2" which probably exists.

This makes total sense then!

I'd drop getCwd(t) for simplicity but it was there before so that's fine.